libvips/resample/thumbnail.c

```diff
@@ -697,17 +703,9 @@ vips_thumbnail_build( VipsObject *object )
 	 */
 	preshrunk_page_height = vips_image_get_page_height( in );
 
-	/* RAD needs special unpacking.
-	 */
-	if( in->Coding == VIPS_CODING_RAD ) {
-		g_info( "unpacking Rad to float" );
-
-		/* rad is scrgb.
-		 */
-		if( vips_rad2float( in, &t[12], NULL ) )
```

Comment:
Is this rad2float conversion no longer required?

Reply:
This was removed since the same thing was done in `colourspace`.
https://github.com/libvips/libvips/blob/65c99219af5b3f4a8e9d4a9daf7886a6c97cf619/libvips/colour/colourspace.c#L556-L563

Though, `colourspace` is not always called after this change, so let me revert this change.